lisa/tools/xfstests.py

```diff
@@ -114,7 +114,8 @@ class Xfstests(Tool):
 
     @property
     def command(self) -> str:
-        return "xfstests"
+        # command is not used in xfstests
```

Comment:
"command is not used in xfstests" it's not true. The command is not directly to be called. But it's used to check if the command installed or not.

Reply:
in this tool, _check_exists is overwrite, I don't think command is used.

Reply:
update